From: 18978327006 <1907810139@qq.com>
Date: Wed, 9 Apr 2025 09:07:25 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 4.5% (7 of 153 strings)

Translation: openkylin-nile-new/peony-vfs-kylin-multiterminal
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/peony-vfs-kylin-multiterminal/vi/
---
 translations/peony-vfs-kylin-multiterminal_vi.ts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/translations/peony-vfs-kylin-multiterminal_vi.ts b/translations/peony-vfs-kylin-multiterminal_vi.ts
index c968cc7..9ecb27d 100644
--- a/translations/peony-vfs-kylin-multiterminal_vi.ts
+++ b/translations/peony-vfs-kylin-multiterminal_vi.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../ui/filemanageview/androidhomepage.cpp" line="14"/>
         <source>Storage</source>
-        <translation type="unfinished"></translation>
+        <translation>Lưu trữ</translation>
     </message>
 </context>
 <context>
@@ -14,17 +14,17 @@
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="69"/>
         <source>Picture</source>
-        <translation type="unfinished"></translation>
+        <translation>HÌnh ảnh</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="72"/>
         <source>Video</source>
-        <translation type="unfinished"></translation>
+        <translation>Video</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="75"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>Âm nhạc</translation>
     </message>
     <message>
         <location filename="../ui/filemanageview/androiditem.cpp" line="78"/>
